test/unit/tech/html5.test.js

```diff
@@ -925,6 +930,19 @@ QUnit.test('When Android Chrome reports Infinity duration with currentTime 0, re
   tech.el_ = oldEl;
 });
 
+QUnit.test('No error is thrown when `load` is called `immediately` after play', function(assert) {
+  const videoTag = TestHelpers.makeTag();
+  const fixture = document.getElementById('qunit-fixture');
+
+  fixture.appendChild(videoTag);
+  const playerInstance = videojs(videoTag.id, { techOrder: ['html5'] });
```

Comment:
we should dispose of this player at the end of the test (`playerInstance.dispose()` after the assertion)

Reply:
Done.